symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkCoreConfig.java

```diff
@@ -74,4 +75,14 @@ public AuthSession botSession(AuthenticatorFactory authenticatorFactory) {
       throw new BeanInitializationException("Unable to authenticate bot", e);
     }
   }
+
+  @Bean
+  @ConditionalOnMissingBean
+  public OboAuthenticator oboAuthenticator(AuthenticatorFactory authenticatorFactory) {
```

Comment:
This bean shouldn't be injected when app is not configured in `application.yaml`